tests/phpunit/tests/functions/deleteoptionfreshsite.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/**
+ * Tests for _delete_option_fresh_site function.
+ *
+ * @group functions.php
+ *
+ * @covers ::_delete_option_fresh_site
+ */
+class Tests_Functions_DeleteOptionFreshSite extends WP_UnitTestCase {
+
+	/**
+	 * @ticket 57191
+	 */
+	public function test__delete_option_fresh_site() {
```

Comment:
```suggestion
	public function test_delete_option_fresh_site() {
```
A double `_` is not convention in the test suite, even for `_private` functions.